packages/teams-js/src/public/pages.ts

```diff
@@ -532,7 +533,7 @@ export namespace pages {
   }
 
   /**
-   * Namespace to interact with the back-stack part of the SDK.
+   * Provides APIs to interact with the back-stack part of the SDK.
```

Comment:
Instead of "back-stack", how about "user's navigational history":

Provides APIs for handling the user's navigational history